src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/Crc32.PlatformNotSupported.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.Arm
+{
+    /// <summary>
+    /// This class provides access to the ARM Crc32 hardware instructions via intrinsics
+    /// </summary>
+    [Intrinsic]
+    [CLSCompliant(false)]
+    public abstract class Crc32 : ArmBase
+    {
+        internal Crc32() { }
+
+        public static new bool IsSupported { [Intrinsic] get { return false; } }
+
+        [Intrinsic]
+        public new abstract class Arm64 : ArmBase.Arm64
+        {
+            internal Arm64() { }
+
+            public static new bool IsSupported { [Intrinsic] get { return false; } }
+
+            /// <summary>
+            /// uint32_t __crc32d (uint32_t a, uint64_t b)
+            ///   A64: CRC32X Wd, Wn, Wm
```

Comment:
Here and in other places the last operand of `CRC32X` should be 'Xm' since it is 64-bit.

Reply:
Yep, missed these two